drltrace/drltrace.config

```diff
@@ -0,0 +1,8 @@
+; The syntax of this config file is simple. To add new function, it's enough to specify
+; each argument of the function, function name itself and return type separated by commas.
```

Comment:
ltrace uses standard C function prototypes e.g. int func_a (int arg1, void *arg2); and it covers almost everything (you can even define your own types and enumerations), however it is more complicated to parse such definition. My first idea was just to use ltrace syntax and don't worry about complexity of the code. Then I decided to use commas. Do you think we will have problems with commas right ? because I don't see any problems at this moment.